.bazelrc

```diff
@@ -18,15 +18,16 @@ build:macos --copt -UDEBUG
 # The number 3G is chosen heuristically to both support large VM and small VM with RBE.
 # Startup options cannot be selected via config.
 # TODO: Adding just to test android
-startup --host_jvm_args=-Xmx3g
+startup --host_jvm_args="-DBAZEL_TRACK_SOURCE_DIRECTORIES=1"
 
 common --noenable_bzlmod
 
 fetch --color=yes
 run --color=yes
 
 build --color=yes
-build --jobs=HOST_CPUS-1
+# Reduced from HOST_CPUS-1 to HOST_CPUS*.5 due to OOMing issues                             # unique
```

Comment:
Is this still needed? 

If so, if we have a CI-specific OOM problem I would try to set those flags in ci/do_ci.sh instead.